frontend/alert/pages/index.tsx

```diff
@@ -132,6 +132,9 @@ function App() {
     useEffect(() => {
         ReactGA.initialize("UA-21029575-12");
         ReactGA.pageview(window.location.pathname + window.location.search);
+        fetch("/api/options/")
```

Comment:
Could we switch this to [platform's URL](https://platform.pennlabs.org/options/)? That way we can toggle all of these in one place

Reply:
Using that URL results in CORS issues. Is CORS enabled for that URL?

Reply:
Ah, that's my bad. I'll push a fix for that in the next day or so